definitions/types/azure-subscription-id.json

```
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "string",
+  "title": "Azure Subscription ID",
+  "description": "Azure Subscription ID",
+  "examples": [
+    "/subscriptions/12345678-1234-1234-abcd-1234567890ab"
+  ],
+  "pattern": "^\/subscriptions\/[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}+$",
```

```suggestion
  "pattern": "^\/subscriptions\/[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$",
```
I don't think we need this trailing `+`